components/Blocks/PlainHero.css

```diff
@@ -16,7 +16,7 @@
   margin: 0 auto 0.5em;
   font-weight: 100;
   line-height: 1.1;
-  width: 31rem;
+  max-width: 31rem;
```

Comment:
Do we deffo 100% for sure want to use `rem`s for max-width? Just thinking how changes in base font size (or the user zooming?) will affect the width of elements and a slight shiver ran down my spine.